dali/pipeline/operators/reader/parser/CMakeLists.txt

```diff
@@ -29,7 +35,12 @@ SET_TARGET_PROPERTIES(CAFFE2_PROTO PROPERTIES COMPILE_FLAGS "-include dali/api_h
 
 # TF
 if (BUILD_PROTO3)
-  protobuf_generate_cpp(TF_PROTO_SRCS TF_PROTO_HEADERS EXPORT_MACRO DLL_PUBLIC proto/example.proto proto/feature.proto)
+  if(${CMAKE_VERSION} VERSION_LESS "3.9.0")
```

Comment:
This way you expose symbols we want to hide. I will address this by providing own protobuf_generate_cpp version shipped with Dali.